src/toorxtreadmill.cpp

```diff
@@ -79,6 +79,9 @@ void toorxtreadmill::update() {
 
     if (initDone) {
         // ******************************************* virtual treadmill init *************************************
+
+        // TODO: determine why no checking if this has been done before
+
         if (!virtualTreadMill) {
```

Comment:
No check of if virtual device setup has been done before
No locksreen/Peloton workaround check.
No lockscreen functionality in this class.

Reply:
yes this module it's not used (at least I guess)